src/ShortUuid.php

```diff
@@ -163,7 +163,7 @@ private function stringToNum(string $string) : BigInteger
     private function setAlphabet(array $alphabet)
     {
         $this->alphabet = $alphabet;
-        $this->alphabetLength = count($alphabet);
+        $this->alphabetLength = BigInteger::of(count($alphabet));
```

Comment:
brick/math converts this value in all operations into BigInteger objects. Now we are doing it here once, so it does not have to be converted again and again.